<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QWidget" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="videoWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="LogoPage">
      <property name="styleSheet">
       <string notr="true">image: url(:/ningePlayer/resource/image/ninge.png);
background-color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QWidget" name="videoPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="Phonon::VideoPlayer" name="videoPlayer"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="controlWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="Phonon::SeekSlider" name="seekSlider">
        <property name="iconVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="play">
          <property name="text">
           <string>Play</string>
          </property>
          <property name="icon">
           <iconset resource="ningePlayerRC.qrc">
            <normaloff>:/ningePlayer/resource/image/media-playback-start.svg</normaloff>:/ningePlayer/resource/image/media-playback-start.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="stop">
          <property name="text">
           <string>Stop</string>
          </property>
          <property name="icon">
           <iconset resource="ningePlayerRC.qrc">
            <normaloff>:/ningePlayer/resource/image/media-playback-stop.svg</normaloff>:/ningePlayer/resource/image/media-playback-stop.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Phonon::VolumeSlider" name="volumeSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="state">
          <property name="text">
           <string>Ready...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="time">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>--:-- / --:--</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ningePlayerRC.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>play</sender>
   <signal>clicked()</signal>
   <receiver>Player</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stop</sender>
   <signal>clicked()</signal>
   <receiver>Player</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>play()</slot>
  <slot>pause()</slot>
  <slot>stop()</slot>
 </slots>
</ui>
